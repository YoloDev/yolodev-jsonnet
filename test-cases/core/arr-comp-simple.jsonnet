[
  x * y
  for x in [1, 2, 3]
  for y in [10, 20, 30]
]
