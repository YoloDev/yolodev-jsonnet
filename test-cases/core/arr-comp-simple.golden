local
    $arr = [
      1,
      2,
      3,
    ];
std["join"](
  [
  ],
  std["makeArray"](
    std["length"](
      $arr,
    ),
      function (
      $i = error "Parameter not bound"
    )
        local
          x = $arr[$i];
        local
          $arr = [
            10,
            20,
            30,
          ];
      std["join"](
        [
        ],
        std["makeArray"](
          std["length"](
            $arr,
          ),
            function (
            $i = error "Parameter not bound"
          )
              local
                y = $arr[$i];
            [
              x
                * y,
            ],
        ),
      ),
  ),
)
